pyomo/repn/tests/ampl/test_ampl_comparison.py

```diff
@@ -98,16 +98,18 @@ def nlwriter_asl_test(self, name):
         pass
 
     # obtain the nl file summary information for comparison with ampl
-    p = pyutilib.subprocess.run(
+    p = subprocess.run(
         'gjh_asl_json '+currdir+name+'.test.nl rows='
-        +currdir+name+'.test.row cols='+currdir+name+'.test.col')
-    self.assertTrue(p[0] == 0, msg=p[1])
+        +currdir+name+'.test.row cols='+currdir+name+'.test.col',
+        stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
+    self.assertTrue(p.returncode == 0, msg=p.stdout.decode("utf-8"))
```

Comment:
I think this can be converted to a list with:
```python
p = subprocess.run(['gjh_asl_json',
    currdir+name+'.test.nl',
    'rows='+currdir+name+'.test.row',
    'cols='+currdir+name+'.test.col'],
    stdout=subprocess.PIPE, stderr=subprocess.PIPE)